projects/kit/components/tiles/tiles.component.ts

```diff
@@ -26,6 +26,7 @@ import {debounce, filter, map} from 'rxjs/operators';
     encapsulation: ViewEncapsulation.None,
     changeDetection: ChangeDetectionStrategy.OnPush,
     providers: [
+        TuiDestroyService,
```

Comment:
we are not injecting this service inside component, is this expected behavior?

Reply:
We are injecting it in `TuiResizeService` and now it is broken because we added `@Self` which exposed this bug.

Reply:
Oh, sorry